### DISCUSSION TITLE
Can use ray tracing SRGs and non-ray tracing SRGs together?

### DISCUSSION THREAD

**Discussion by ohmaya** [o3de/o3de#14308] [Help Wanted]
As shown in https://github.com/o3de/o3de/discussions/14074, it is possible to use both RT SRGs and non-RT SRGs together?

There is a comment:
// Copy of ViewSrg::m_viewProjectionInverseMatrix since we can't access ViewSrg in ray tracing shaders
ATM.

What if FrequencyId's conflict (both have 0, 1, and 2)?

**Comment by ZhuChasel** [comment]
i have the same feature...

**Reply by ohmaya** [reply]
Maybe you mean, you encountered the same issue.
